decision_service/include/constants.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+namespace reinforcement_learning {
+  using str_const = const char * const;
```

Comment:
>using str_const = const char * const; [](start = 2, length = 37)

why not just use the type below?

Reply:
I wanted to declare str_const in the reinforcement_learning namespace and use it in reinforcement_learning::name and reinforcement_learning::value namespaces.  Not sure if that is your question.

---
In reply to: [193405380](https://github.com/JohnLangford/vowpal_wabbit/pull/1495#discussion_r193405380) [](ancestors = 193405380)

Reply:
I was questioning if the typedef makes it more readable... maybe if it is that close, it's fine... 

---
In reply to: [193425684](https://github.com/JohnLangford/vowpal_wabbit/pull/1495#discussion_r193425684) [](ancestors = 193425684,193405380)

Reply:
Done

---
In reply to: [193484325](https://github.com/JohnLangford/vowpal_wabbit/pull/1495#discussion_r193484325) [](ancestors = 193484325,193425684,193405380)